deployment/view/common_test.go

```diff
@@ -55,7 +57,7 @@ func TestParseExecutionResultForDestChainSelectors(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got := ParseExecutionResultForDestChainSelectors(tt.input)
+			got := common.ParseExecutionResultForDestChainSelectors(tt.input)
```

Comment:
This func is not CCIP specific or DestChainSelectors specific.

Let's rename and remove from `pkg/ccip/bindings` into something more TON general, and then use it for CCIP binding.